examples/bbox_detection/labelme2voc.py

```diff
@@ -101,7 +106,19 @@ def main():
             class_id = class_names.index(class_name)
 
             (xmin, ymin), (xmax, ymax) = shape['points']
-            bboxes.append([xmin, ymin, xmax, ymax])
+            
+            # swap if min is larger than max.
+            if xmin > xmax:
+                tmp = xmin
+                xmin = xmax
+                xmax = tmp
+            if ymin > ymax:
+                tmp = ymin
+                ymin = ymax
+                ymax = tmp
```

Comment:
Please write in more pythonic way. e.g., `xmin = min(xmin, xmax)`

Reply:
yes, rewrote with sort.
```
xmin, xmax = sorted([xmin, xmax])
ymin, ymax = sorted([ymin, ymax])
```